src/prommis/nanofiltration/diafiltration_solute_properties.py

```diff
@@ -0,0 +1,147 @@
+#####################################################################################################
+# “PrOMMiS” was produced under the DOE Process Optimization and Modeling for Minerals Sustainability
+# (“PrOMMiS”) initiative, and is copyright (c) 2023-2025 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory, et al. All rights reserved.
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license information.
+#####################################################################################################
+"""
+Property package for the diafiltration cascade.
+
+Author: Molly Dougher
+"""
+
+from pyomo.environ import Param, Var, units
+
+from idaes.core import (
+    Component,
+    MaterialFlowBasis,
+    PhysicalParameterBlock,
+    StateBlock,
+    StateBlockData,
+    declare_process_block_class,
+)
+from idaes.core.util.initialization import fix_state_vars
+
+
+@declare_process_block_class("SoluteParameter")
+class SoluteParameterData(PhysicalParameterBlock):
+    """
+    Property Package for the diafiltration cascade.
+
+    Currently includes the following solutes:
+        Li+ (lithium ion)
+        Co2+ (cobalt ion)
+        Cl- (chlorine ion)
+    """
+
+    def build(self):
+        super().build()
+
+        # add cations
+        self.Li = Component()
+        self.Co = Component()
+
+        # add anions
+        self.Cl = Component()
+
+        # add valence
+        self.charge = Param(
+            self.component_list,
+            units=units.dimensionless,
+            initialize={
+                "Li": 1,
+                "Co": 2,
+                "Cl": -1,
+            },
+        )
+
+        # add molecular weight
+        self.molar_mass = Param(
+            self.component_list,
+            units=units.kg / units.mol,
+            initialize={
+                "Li": 0.006941,
+                "Co": 0.05893,
+                "Cl": 0.03545,
+            },
+        )
+
+        # add sigma
```

Comment:
Maybe change this to "add osmotic coefficient, where 1 represents ideal behavior"?